inc/template.php

```diff
@@ -1018,7 +1018,7 @@ function tpl_img_getTag($tags, $alt = '', $src = null) {
     if(is_null($meta)) $meta = new JpegMeta($src);
     if($meta === false) return $alt;
     $info = cleanText($meta->getField($tags));
-    $meta = null; // garbage collect and close any file handles. See #3404
+    // $meta = null; // garbage collect and close any file handles. See #3404
```

Comment:
This undoes 01c53a65419273208ff4c5e111338e35f88c1394 without an alternative way of fixing the problem?

Reply:
I apologize, I assumed it was going to work correctly since I didn't see any errors on my system. That was incorrect and I should have run git blame on this before changing it like that. I've reverted this commit and the new solution should fix both problems.